processes/consumer/process.go

```diff
@@ -51,8 +51,7 @@ func (p processArgs) process(ctx context.Context, cfg config.Config, inMemDB *mo
 		return "", fmt.Errorf("cannot unmarshall key %s: %w", string(p.Msg.Key()), err)
 	}
 
-	typingSettings := cfg.SharedTransferConfig.TypingSettings
-	_event, err := topicConfig.GetEventFromBytes(typingSettings, p.Msg.Value())
+	_event, err := topicConfig.GetEventFromBytes(p.Msg.Value())
```

Comment:
Could change `process` to accept `cfg.Mode` instead of `cfg` now.

Reply:
Ah, we still need it for event save

```go
	shouldFlush, flushReason, err := evt.Save(cfg, inMemDB, topicConfig.tc, p.Msg)
	if err != nil {
		tags["what"] = "save_fail"
		return "", fmt.Errorf("event failed to save: %w", err)
	}
```

Reply:
Ah missed that!